arch/i386/mcount-support.c

```diff
@@ -211,7 +211,8 @@ unsigned long *mcount_arch_parent_location(struct symtabs *symtabs,
 		if (!(strcmp(find_main[0], parent_name) || 
 		      strcmp(find_main[1], child_name))) {
 			ret_addr = *parent_loc;
-			for (int i = 1; i < MAX_SEARCH_STACK; i++ ) {
+			int i;
+			for (i = 1; i < MAX_SEARCH_STACK; i++) {
```

Comment:
I prefer having declarations before statements - I know it's an old style but...